bitnami/mysql/README.md

```diff
@@ -5,7 +5,7 @@
 ## TL;DR;
 
 ```bash
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
 $ helm install my-release bitnami/mysql
```

Comment:
mysql is not in the _stable_ repo so those instructions doesn't work:
```
▶ helm repo list
Error: no repositories to show

▶ helm repo add stable https://kubernetes-charts.storage.googleapis.com
”stable” has been added to your repositories

▶ helm install my-release bitnami/mysql
Error: failed to download “bitnami/mysql” (hint: running `helm repo update` may help)
```

Adding the _bitnami_ repo it works as expected:
```
▶ helm repo add bitnami https://charts.bitnami.com/bitnami
“bitnami” has been added to your repositories

▶ helm install my-release bitnami/mysql
NAME: my-release
LAST DEPLOYED: Mon Feb 17 16:08:10 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Please be patient while the chart is being deployed

Tip:

  Watch the deployment status using the command: kubectl get pods -w --namespace default

Services:

  echo Master: my-release-mysql.default.svc.cluster.local:3306
  echo Slave:  my-release-mysql-slave.default.svc.cluster.local:3306

Administrator credentials:

  echo Username: root
  echo Password : $(kubectl get secret --namespace default my-release-mysql -o jsonpath=“{.data.mysql-root-password}” | base64 --decode)

To connect to your database:

  1. Run a pod that you can use as a client:

      kubectl run my-release-mysql-client --rm --tty -i --restart=‘Never’ --image  docker.io/bitnami/mysql:8.0.19-debian-10-r0 --namespace default --command -- bash

  2. To connect to master service (read/write):

      mysql -h my-release-mysql.default.svc.cluster.local -uroot -p my_database

  3. To connect to slave service (read-only):

      mysql -h my-release-mysql-slave.default.svc.cluster.local -uroot -p my_database

To upgrade this helm chart:

  1. Obtain the password as described on the ‘Administrator credentials’ section and set the ‘root.password’ parameter as shown below:

      ROOT_PASSWORD=$(kubectl get secret --namespace default my-release-mysql -o jsonpath=“{.data.mysql-root-password}” | base64 --decode)
      helm upgrade my-release bitnami/mysql --set root.password=$ROOT_PASSWORD
```